src/pages/ReportDetailsPage.tsx

```diff
@@ -218,6 +225,13 @@ function ReportDetailsPage({policies, report, session, personalDetails}: ReportD
         Report.leaveGroupChat(report.reportID);
     }, [isChatRoom, isPolicyEmployee, isPolicyExpenseChat, report.reportID, report.visibility]);
 
+    const unapproveExpenseReportOrShowModal = useCallback(() => {
+        // TODO: show modal if report is exported to accounting
+        // setIsUnapproveModalVisible(true);
+
+        IOU.unapproveExpenseReport(moneyRequestReport ?? {});
```

Comment:
maaaybe we don't need to default `?? {}` here anymore since we handle that in the function?